WeScan/Common/CIRectangleDetector.swift

```diff
@@ -11,7 +11,7 @@ import AVFoundation
 
 /// Class used to detect rectangles from an image.
 struct CIRectangleDetector {
-  
+    
```

Comment:
What happened with the indentation in this file?

Reply:
The indentation was previously incorrect to the rest of WeScan, so Xcode fixed it (to 4 spaces instead of 2)